### PR TITLE
Wrap OGR_L_GetSpatialRef, OSRAutoIdentifyEPSG, fix return type

### PR DESCRIPTION
**Pull request by lnicola**

### CODE REVIEW

File: gdal-sys/src/ogr_enums.rs [hunk 1]
```diff
@@ -36,3 +36,6 @@ pub enum OGRFieldType {
     OFTInteger64List = 13,
     OFTMaxType = 14
 }
+
+pub const FALSE: i32 = 0;
+pub const TRUE: i32 = 1;
```
Comment (jdroenner) [inline gdal-sys/src/ogr_enums.rs]: Is the `i32 `always correct or is this a `c_int`?
Maybe we cloud call it `C_TRUE `and `C_FALSE` ?

Comment (lnicola) [reply gdal-sys/src/ogr_enums.rs]: Good point. Bikeshed: maybe `OGR_TRUE`?

Comment (jdroenner) [reply gdal-sys/src/ogr_enums.rs]: well it's not really a OGR constant... So i would prefer the C prefix.

Comment (lnicola) [reply gdal-sys/src/ogr_enums.rs]: `CPL_` would have been the best :smile:

Comment (jdroenner) [reply gdal-sys/src/ogr_enums.rs]: GDAL prefixes are a bit confusing sometimes 😄

**Comment by lnicola** [comment]
@jdroenner Can we mergethis as-is?

**Pull request merged**